help/status-and-availability.md

```diff
@@ -50,10 +50,34 @@ You can set a status emoji, status message, or both.
 Status emoji are shown next to a user's name in the sidebars, message feed,
 and compose box.
 
-You can view status messages by hovering over your or anyone else's name in the
-left or right sidebar, or by clicking the user's name or avatar in the main
-message feed to open their **user card**. If someone hasn't set a message as
-part of their status, then no status message will appear.
+{start_tabs}
+
+{tab|desktop-web}
+
+1. Hover over a user's name in the right sidebar.
+
+1. Click on the **ellipsis** (<i class="zulip-icon zulip-icon-ellipsis-v-solid"></i>)
+   to the right of their name to open their **user card**. You can also click on
+   a user's profile picture or name on a message they sent.
+
+!!! tip ""
+
+    If someone hasn't set a message as part of their status, then no status
+    message will appear.
+
+{tab|mobile}
+
+1. Tap on a user's profile picture or name. If you are viewing a private message
+   conversation, you can also tap the **information**
+   (<img src="/static/images/help/mobile-info-circle-icon.svg" alt="information" class="mobile-icon"/>)
+   button in the top right corner of the app.
+
```

Comment:
I think we should add some instruction about where to look in the profile, since there's a lot going on there, and the status isn't explicitly labeled. Maybe say that it's below their role (e.g., Member)?